tests/src/e2e/setup.rs

```diff
@@ -420,7 +421,7 @@ impl Test {
                 .join(self.net.chain_id.as_str())
                 .join(utils::NET_ACCOUNTS_DIR)
                 .join(format!("validator-{}", index))
-                .join(config::DEFAULT_BASE_DIR),
+                .join(default_namada_folder),
```

Comment:
This won't work as `default_namada_folder` is an absolute path and so with `.`join` it overrides the path in full. We could probably keep using `.namada` for e2e tests, or perhaps better just use the `"validator-{}"` as a base dir